blockchain/chain/src/store/chain_store.rs

```diff
@@ -967,20 +967,17 @@ where
 
 pub async fn sub_head_changes(
     mut subscribed_head_change: Subscriber<HeadChange>,
-    heaviest_tipset: &Option<Arc<Tipset>>,
+    heaviest_tipset: &Arc<Tipset>,
```

Comment:
also, does it need to be a reference if it can just be moved? it's cloned during the subscribe?